servant/src/Servant/API/MultiVerb.hs

```diff
@@ -146,6 +149,11 @@ instance AsHeaders '[h] a (a, h) where
   toHeaders (t, cc) = (I cc :* Nil, t)
   fromHeaders (I cc :* Nil, t) = (t, cc)
 
+-- | Two headers and an empty response, return value is a tuple of the response and the header
```

Comment:
i don't understand this comment.  return value is the tuple of the two headers, no?  response doesn't need to be returned because it's ()?